content/installation/administrator/admin_install_r/Rprofile.site

```diff
@@ -36,6 +35,3 @@ if (interactive()) {
     }
   )
 }
-
-# required for RStan and brms
-Sys.setenv(BINPREF = "C:/Rtools/mingw_$(WIN)/bin/")
```

Comment:
So RStan and brms no longer need to know where Rtools is located?

Reply:
Not exactly... they do need to know this, but I think it solved by following the installation instructions provided by https://cran.r-project.org/bin/windows/Rtools/. See section Putting Rtools on the PATH.